packages/calypso-e2e/src/lib/pages/media-page.ts

```diff
@@ -19,6 +24,8 @@ const selectors = {
 	items: '.media-library__list-item',
 	placeholder: '.is-placeholder',
 	editButton: 'button[data-e2e-button="edit"]',
+	addNewButton: '.media-library__upload-button-input',
```

Comment:
Can we select based on the `Add new` text?

Reply:
Does not look like this can be done.

The `Add new` text is a `span` and is a sibling to the input. 
Since we are not clicking on the `Add new` button, selecting by the `Add new` text wouldn't work here:

```
 1) [WPCOM] Media: Upload: (desktop) @parallel
       Upload media files (Simple)
         Upload image and confirm addition to gallery:
     page.setInputFiles: Node is not an HTMLInputElement
=========================== logs ===========================
waiting for selector "span:text("Add new")"
  selector resolved to hidden <span data-e2e-sidebar="Add New" class="sidebar__menu-…>Add New</span>
```